apisix/init.lua

```diff
@@ -116,18 +116,18 @@ function _M.http_init_worker()
 
     require("apisix.debug").init_worker()
 
+    if core.config == require("apisix.core.config_xds") then
```

Comment:
Can we just put all the init_worker together?

Reply:
For `config_xds`, I need it to run before route, service is initialised. So do I need to advance `config_yaml` to the same level as `config_xds`?

Reply:
Yes.

Reply:
fix